cds/modules/theme/static/js/cds/module.js

```diff
@@ -212,20 +212,65 @@ app.filter('findResolution', function($filter) {
   return function(record) {
     height = record['tags']['height'];
     selectedResolution = height.concat('p');
+
+    configuredResolutionsWidth = {
+      '1920': '1080p',
+      '2048': '2K'
+    }
+
+    configuredResolutions = {
+      '240': '240p',
+      '360': '360p',
+      '480': '480p',
+      '720': '720p',
+      '1024': '1024p',
+      '1080': '1080p',
+      '1080': '2K',
+      '2160': '4K',
+      '4320': '8K'
+    }
+
+    Object.keys(configuredResolutions).forEach(function(resolution) {
+      if (height >= resolution) {
+        selectedResolution = configuredResolutions[resolution]
```

Comment:
Missing `;`. Also I had the same question why `findResolution` is defined twice and with different configured resolutions.